ros/ros1/OdometryServer.cpp

```diff
@@ -76,101 +75,97 @@ OdometryServer::OdometryServer(const ros::NodeHandle &nh, const ros::NodeHandle
                                                               &OdometryServer::RegisterFrame, this);
 
     // Initialize publishers
-    odom_publisher_ = pnh_.advertise<nav_msgs::Odometry>("odometry", queue_size_);
-    frame_publisher_ = pnh_.advertise<sensor_msgs::PointCloud2>("frame", queue_size_);
-    kpoints_publisher_ = pnh_.advertise<sensor_msgs::PointCloud2>("keypoints", queue_size_);
-    map_publisher_ = pnh_.advertise<sensor_msgs::PointCloud2>("local_map", queue_size_);
-
-    // Initialize trajectory publisher
+    odom_publisher_ = pnh_.advertise<nav_msgs::Odometry>("/kiss/odometry", queue_size_);
+    frame_publisher_ = pnh_.advertise<sensor_msgs::PointCloud2>("/kiss/frame", queue_size_);
+    kpoints_publisher_ = pnh_.advertise<sensor_msgs::PointCloud2>("/kiss/keypoints", queue_size_);
```

Comment:
I got rid of these debugging publishers in the ROS 2 wrapper. To avoid doing additional transformations just for visualization. At the end, the frame is "identical" (if you don't deksew). And I think the python API is better anyways for such debugging scenarios...

Reply:
Nice catch! Yes, I accidentally left these publishers in even though they are no longer used.